pkg/apis/serving/register.go

```diff
@@ -20,6 +20,9 @@ const (
 	// GroupName is the group name for knative labels and annotations
 	GroupName = "serving.knative.dev"
 
+	// GroupNamePrefix is the prefix for label key and annotation key
+	GroupNamePrefix = GroupName + "/"
```

Comment:
pkg/apis/serving/v1beta1/register.go: has this already.
Can we re-use?

Reply:
I deleted the entry in `pkg/apis/serving/v1beta1/register.go` and declared it once here.